### ISSUE TITLE
Rename addComponentExample and addScreenExample.

### ISSUE THREAD

**Issue by kevinvangelder** [infinitered/ignite#636]
When I use `ignite.addComponentExample` I expect the example to appear in the Component Examples screen. However, since we're running this from a plugin it's really adding it to the Plugin Examples screen. We should do something to make it more clear what's happening.

**Comment by GantMan** [comment]
Do you recommend `addPluginComponentExample` or too wordy?

**Comment by kevinvangelder** [comment]
That would at least be a step in the right direction. Since it will only ever be used by Plugin developers I don't think that's too verbose.

**Comment by skellock** [comment]
I like wordy.

**Comment by jamonholmgren** [comment]
@robinheinze Add a pleasant error for those using `addComponentExample` so they know to switch to `addPluginComponentExample`.

**Comment by skellock** [comment]
🚨 SCOPE CREEP ALERT 🚨 

Can you update `addScreenExample` to become `addPluginScreenExample` as well?  ❤️

**Comment by robinheinze** [comment]
Yup, will do
